### DISCUSSION TITLE
Space before heading

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#44027] [Mobile] [answered]
Visit https://github.com/dart-lang/sdk

GitHub Mobile beta:

![Screenshot_20230111-172406.png](https://user-images.githubusercontent.com/2119212/211930923-4358b931-a37a-43c6-9d4d-d7f3a4fb6741.png)

Note the leading space before `A client-optimized language for fast apps on any platform`

Apparently it's for some hover effect:
![Screenshot_20230112-190007.png](https://user-images.githubusercontent.com/2119212/212206407-67a0b865-860d-4dc3-a10c-0cd33b3be789.png)
![Screenshot_20230112-190023.png](https://user-images.githubusercontent.com/2119212/212206449-c2b6c163-f791-4b7b-a4a6-78bf64d25bb1.png)
-- which really doesn't look great. No idea what it is. If this it the goal, I'd recommend not using hanging indentation on mobile. Sticking a thing above the row of text or on the far side of the line (in RtL that might not be the right side, although I haven't checked if that's supported...).

Chrome Android:
![Screenshot_20230111-172524.png](https://user-images.githubusercontent.com/2119212/211931079-a01a49f4-9ae6-4865-a1a8-6c28e95993b4.png)

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref thanks for filing. I passed this onto the team for prioritization.

**Comment by alcere** [comment]
Hi @jsoref! We have solved this issue, and today's release (1.141.0) will contain the fix. Thanks for reporting!
